Corona-Warn-App/src/main/res/values-bg/dcc_ticketing_strings.xml

```diff
@@ -71,7 +71,7 @@
     <!--XTXT: Certificate Selection Screen: FAQ Link DE -->
     <string name="dcc_ticketing_faq_link_german">"https://www.coronawarn.app/de/faq/#val_service_no_valid_dcc"</string>
     <!--XTXT: Certificate Selection Screen: FAQ Link EN -->
-    <string name="dcc_ticketing_faq_link_english">"https://www.coronawarn.app/en/faq/#val_service_no_valid_dcc"</string>
+    <string name="dcc_ticketing_faq_link_english">"https://wwsw.coronawarn.app/en/faq/#val_service_no_valid_dcc"</string>
```

Comment:
Careful, reverts the changes of #4488

Reply:
@KathrinLuecke @CV113 could you take care of this? It should be `www` in all languages instead of `wwsw`.

Reply:
@marcauberer  I merged this PR into https://github.com/corona-warn-app/cwa-app-android/pull/4500 to fix this issue 
Please have a look ?